src/pages/learn/schema.mdx

```diff
@@ -20,17 +26,19 @@ If you've seen a GraphQL query before, you know that the GraphQL query language
 2. We select the `hero` field on that
 3. For the object returned by `hero`, we select the `name` and `appearsIn` fields
 
-Because the shape of a GraphQL query closely matches the result, you can predict what the query will return without knowing that much about the server. But it's useful to have an exact description of the data we can ask for - what fields can we select? What kinds of objects might they return? What fields are available on those sub-objects? That's where the schema comes in.
+Because the shape of a GraphQL query closely matches the result, we can predict what the query will return without knowing that much about the server. But it's useful to have an exact description of the data we can request. For example, what fields can we select? What kinds of objects might they return? What fields are available on those sub-objects?
 
-Every GraphQL service defines a set of types which completely describe the set of possible data you can query on that service. Then, when queries come in, they are validated and executed against that schema.
+That's where the schema comes in. Every GraphQL service defines a set of types that completely describe the set of possible data we can query on that service. Then, when requests come in, they are validated and executed against that schema.
 
 ## Type language
 
-GraphQL services can be written in any language. Since we can't rely on a specific programming language syntax, like JavaScript, to talk about GraphQL schemas, we'll define our own simple language. We'll use the "GraphQL schema language" - it's similar to the query language, and allows us to talk about GraphQL schemas in a language-agnostic way.
+GraphQL services can be written in any language and there are different approaches you can take when defining the types in a schema. Some libraries require that you only write the _resolver functions_ that provide data for the different fields in the API, then compile the schema based on that code. Other libraries allow you to define types and fields using what's commonly called the _schema definition language_ (or SDL) and then write the resolver functions for the corresponding fields separately.
+
+Since we can't rely on a specific programming language to discuss GraphQL schemas in this guide, we'll use SDL because it's similar to the query language that we've seen so far and allows us to talk about GraphQL schemas in a language-agnostic way.
 
 ## Object types and fields
 
-The most basic components of a GraphQL schema are object types, which just represent a kind of object you can fetch from your service, and what fields it has. In the GraphQL schema language, we might represent it like this:
+The most basic components of a GraphQL schema are [Object types](https://spec.graphql.org/draft/#sec-Objects), which just represent a kind of object you can fetch from your service, and what fields it has. In SDL, we represent it like this:
```

Comment:
I was on the fence about using "SDL" for this reason but I ended up introducing it on this page because it is so ubiquitous outside of the spec, so I thought it might be best for clarity's sake. Do you think it would be helpful to indicate that this is a common term (not a spec term) or is that too much information at this point?

Reply:
I think what you have is fine, truly this was just trivia. Thank you for thinking about it! (Perhaps the first time we use the term "SDL" we could says something like "colloquially known as SDL"? No big deal either way.)